.meeseeksdev.yml

```diff
@@ -0,0 +1,12 @@
+users:
+  hmaarrfk:
+    can:
+      - backport
+      - tag
+    config:
+      tag:
+        any: False
+        only:
+           - 'backport: 0.14.x'
+
```

Comment:
Pep8: only 1 blank line at the end of the file :D

Reply:
Pretty sure PEP8 doesn't cover .yml files... ;)